demos/custom-tabs-example-app/src/main/AndroidManifest.xml

```diff
@@ -10,70 +11,83 @@
      See the License for the specific language governing permissions and
      limitations under the License.
 -->
-
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.chromium.customtabsdemos">
 
+    <queries>
+        <intent>
+            <action android:name="android.support.customtabs.action.CustomTabsService" />
+        </intent>
+    </queries>
+
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/Theme.AppCompat.Light.DarkActionBar">
-
-        <activity android:name=".DemoListActivity"
-            android:label="@string/app_name"
-            android:exported="true">
+        <activity
```

Comment:
nit: Put this below `DemoListActivity` - that's the first Activity people will see.

Reply:
Done.